### PR TITLE
Add SmartParse

### PR DESCRIPTION
**Pull request by ivanbreet**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,6 +339,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Shipyard](https://www.shipyardapp.com) — Low-code data orchestration platform for the cloud. Build with a mix of low-code templates and your code (Python, Node.js, Bash, SQL). Our free developer plan offers 10 hours of runtime every month for one user - more than enough to automate multiple workflows.
   * [shrtcode API](https://shrtco.de/docs) - Free URL Shortening API without authorization and no request limits.
   * [SerpApi](https://serpapi.com/) - Real-time search engine scraping API. Returns structured JSON results for Google, YouTube, Bing, Baidu, Walmart, and many other machines. The free plan includes 100 successful API calls per month.
+  * [SmartParse](https://smartparse.io) - SmartParse is a data migration and CSV to API platform that offers time- and cost-saving developer tools. The Free tier includes: Up to 300 processing units, Browser uploads, Data quarantining, Circuit breakers and Job Alerts.
```
Comment (ripienaar) [inline README.md:342]: Is this 300 units per month or a one-off?

Comment (ivanbreet) [reply README.md:342]: The Free tier is limited to 300 units every month.

Comment (ripienaar) [reply README.md:342]: please clarify that in the PR

**Comment by ivanbreet** [comment]
Updated perks 👍🏻

**Comment by ripienaar** [comment]
thanks

**Pull request merged**